accounts/admin.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from players.models import Player
+
+from .models import User
+
+
+class PlayerInline(admin.StackedInline):
+    model = Player
+
+
+class UserAdmin(UserAdmin):
+    # Copied all of these from `UserAdmin` and replaced all instances of `username` with `email`.
+    fieldsets = (
+        (None, {"fields": ("email", "password")}),
+        ("Personal info", {"fields": ("first_name", "last_name")}),
+        (
+            "Permissions",
+            {
+                "fields": (
+                    "is_active",
+                    "is_staff",
+                    "is_superuser",
+                    "groups",
+                    "user_permissions",
+                ),
+            },
+        ),
+        ("Important dates", {"fields": ("last_login", "date_joined")}),
+    )
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
+                "fields": ("email", "password1", "password2"),
+            },
+        ),
+    )
+    list_display = ("email", "first_name", "last_name", "is_staff")
+    list_filter = ("is_staff", "is_superuser", "is_active", "groups")
+    search_fields = ("first_name", "last_name", "email")
+    ordering = ("email",)
+
+    inlines = [PlayerInline]
```

Comment:
This is the only clever thing that wasn't copied from somewhere. This means when we create/edit a user via django admin, we also have access to create/edit the player model (Specifically the handle).